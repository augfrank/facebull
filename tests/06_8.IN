M0   C0   C2  2474
M1   C0   C3  2477
M2   C0   C4  2760
M3   C0   C5  2516
M4   C1   C0  2763
M5   C1   C2  2693
M6   C1   C3  2552
M7   C1   C4  2593
M8   C1   C5  2885
M9   C2   C0  2668
M10   C2   C1  2522
M11   C2   C4  2522
M12   C2   C5  2883
M13   C3   C0  2565
M14   C3   C1  2962
M15   C3   C2  2758
M16   C3   C4  2093
M17   C3   C5  2357
M18   C4   C0  2311
M19   C4   C1  2254
M20   C4   C2  2810
M21   C4   C3  2126
M22   C4   C5  2020
M23   C5   C0  2697
M24   C5   C1  2426
M25   C5   C2  2868
M26   C5   C3  2620
