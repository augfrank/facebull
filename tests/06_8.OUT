14358
0 6 10 16 22 23
